libs/formats/src/zone/zone-util.ts

```diff
@@ -0,0 +1,7 @@
+import { MapZones } from '@momentum/constants';
+
+export const ZoneUtil = {
+  isLinearMainTrack: function (zoneData: MapZones): boolean {
+    return zoneData.tracks.main.zones.segments.length === 1;
```

Comment:
Is this the right check? A linear track is a track with just minor checkpoints. This seems to be checking if it just has one singular segment? What if there's multiple minor checkpoints? Or, I suppose, what are "segments" in this context? Our doc where we drafted this only has tracks, zones, volumes, and regions.

Reply:
Ah, I moved this commit quite high as I wanted to get through the formatting lib changes first, but I suppose it means this code ends up before the types for MapZones is given. See constants/models/maps/MapZones.ts 

The current zone file spec uses "segments" and "checkpoints", for major and minor checkpoints respectively, I don't mind refactoring but we'd have to do here and in C++ ASAP. rio brought this up the terminology briefly in mom-dev red on 26/10

Reply:
@Gocnak The original doc wasn't necessarily totally precise yet and was generally still structured from a relational DB point of view. Now that we know this data is all JSON, our data can be more structured, and as we implemented things we realized grouping zones into "segments" each with a list of "checkpoints" fit the needs of the data and has multiple benefits. The segment number is the "major number" and the checkpoint number within the segment is the "minor number". The first "checkpoint" in a segment is a "major checkpoint" in the old terminology (aka the track start if it is the first segment). And yeah if the Main track is linear, it has just one segment.